hydrabus_framework/utils/logger.py

```diff
@@ -0,0 +1,106 @@
+from tabulate import tabulate
+
+from hydrabus_framework.utils.color import Colors
+
+
+__author__ = "Jordan Ovrè <ghecko78@gmail.com>"
+
+
+class Logger:
+    """
+    The aim of this class is to manage core framework and module printing
+    """
+    ERROR = 1
+    SUCCESS = 2
+    INFO = 3
+    RESULT = 4
+    USER_INTERACT = 5
+    HEADER = 6
+
+    def __init__(self):
+        self.categories = [
+            {"category": 1, "fct": self._print_error},
+            {"category": 2, "fct": self._print_success},
+            {"category": 3, "fct": self._print_info},
+            {"category": 4, "fct": self._print_result},
+            {"category": 5, "fct": self._print_user_interact},
+            {"category": 6, "fct": self._print_header},
+        ]
+
+    def print(self, text, category=None):
```

Comment:
print is already defined in python3 default packages (cf. https://docs.python.org/3.0/whatsnew/3.0.html). You can use a generic term like handle

Reply:
Done [#8bdd2cc](https://github.com/hydrabus-framework/framework/commit/8bdd2cce8c1726552edbc7aac2a06300c56f80ae)